djstripe/migrations/0019_alter_payout_destination_clone.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.12 on 2022-03-29 06:49
+
+import django.db.models.deletion
+from django.db import migrations
+
+import djstripe.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("djstripe", "0018_add_payout_destination_clone"),
+    ]
+
+    operations = [
+        # populate values in destination_clone model field
+        migrations.RunSQL(
+            sql="""
+            DO $$
+            DECLARE
+            destination_pk varchar;
+            BEGIN
+            FOR destination_pk IN SELECT destination_id FROM djstripe_payout
+            LOOP
+                UPDATE djstripe_payout SET destination_clone_id = (SELECT id FROM djstripe_bankaccount WHERE djstripe_id::VARCHAR = destination_pk);
+            END LOOP;
+            END$$;
```

Comment:
```suggestion
UPDATE djstripe_payout SET destination_clone_id = (select id from djstripe_bankaccount where djstripe_bankaccount.djstripe_id = djstripe_payout.destination_id)
WHERE EXISTS(select * from djstripe_bankaccount where djstripe_bankaccount.djstripe_id = djstripe_payout.destination_id);
```

I'm not sure if that SQL will run on all engines. I did a bit of research to find the ANSI SQL syntax for an UPDATE that effectively joins another table. This is what I came up with.

I've tested on postgres and sqlite.